keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/ec2.kt

```diff
@@ -16,6 +16,19 @@
 package com.netflix.spinnaker.keel.ec2
 
 import com.netflix.spinnaker.keel.api.ApiVersion
+import com.netflix.spinnaker.keel.api.ec2.ApplicationLoadBalancerSpec
+import com.netflix.spinnaker.keel.api.ec2.ClassicLoadBalancerSpec
+import com.netflix.spinnaker.keel.api.ec2.ClusterSpec
+import com.netflix.spinnaker.keel.api.ec2.SecurityGroupSpec
+import com.netflix.spinnaker.keel.api.plugins.kind
 
 val SPINNAKER_EC2_API_V1 = ApiVersion("ec2")
 const val CLOUD_PROVIDER = "aws"
+
+val CLUSTER_V1 = kind<ClusterSpec>("ec2/cluster@v1")
+
+val SECURITY_GROUP_V1 = kind<SecurityGroupSpec>("ec2/security-group@v1")
+
+val CLASSIC_LOAD_BALANCER_V1 = kind<ClassicLoadBalancerSpec>("ec2/classic-load-balancer@v1")
+
+val APPLICATION_LOAD_BALANCER_V1 = kind<ApplicationLoadBalancerSpec>("ec2/application-load-balancer@v1")
```

Comment:
Made constants for all the resource kinds. These are exported as beans in `EC2Config` so we no longer rely on `ResourceHandler` implementations to work out what kinds we support. That means we can have kinds with no associated handler (e.g. the handler supports the latest version, but we still have the class for the older versions).